content/en/docs/setup/best-practices/multiple-zones.md

```diff
@@ -95,13 +95,15 @@ such as Deployment, StatefulSet, or Job.
 
 ## Storage access for zones
 
-When persistent volumes are created, the `PersistentVolumeLabel`
-[admission controller](/docs/reference/access-authn-authz/admission-controllers/)
-automatically adds zone labels to any PersistentVolumes that are linked to a specific
-zone. The {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} then ensures,
+When persistent volumes are created, Kubernetes automatically adds zone labels 
+to any PersistentVolumes that are linked to a specific zone. The {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} then ensures,
```

Comment:
nit
```suggestion
to any PersistentVolumes that are linked to a specific zone.
The {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} then ensures,
```

Reply:
@T-Lakshmi  what do yo mean by "nit" ?

Reply:
@hassanbenhzaine,
My intention here is to request that you rework a minor flaw that I identified.

Reply:
@T-Lakshmi I fixed the issue, please review the code :)

Reply:
@hassanbenhzaine,
Thanks!